frontend/src/components/AdminPanel/adminApi.tsx

```diff
@@ -146,7 +147,7 @@ const listGroupLeadersInOrganization = async (organizationID: string) =>
 const listGroupLeaders = async () => await listUsersInGroup('groupLeader')
 const listAdminsInOrganization = async (organizationID: string) =>
   await listUsersInGroup(`${organizationID}${ADMIN_COGNITOGROUP_SUFFIX}`)
-const listAdmins = async () => await listUsersInGroup('admin')
+const listAdmins = async () => await listUsersInGroup(SUPER_ADMIN_COGNITO_GROUP)
```

Comment:
Burde kanskje bytte navn på denne til listSuperAdmins (bare for å være mer konsistent)